modules/cert-manager-override-scheduling.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-override-scheduling_{context}"]
+= Configuring Scheduling Overrides for {cert-manager} Components
+
+After installing the {cert-manager-operator}, you can configure the scheduling of {cert-manager} components by configuring the `overrideScheduling` field in the {cert-manager} custom resource. This allows you to define `nodeSelector` and `tolerations` for the `controllerConfig`, `webhookConfig`, and `cainjectorConfig` sections.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have installed the {cert-manager-operator} 1.12.0 or later.
+
+.Procedure
+
+* To configure scheduling overrides, modify the {cert-manager} custom resource. Use the `overrideScheduling` field within `controllerConfig`, `webhookConfig`, or `cainjectorConfig` to specify `nodeSelector` and `tolerations` for the respective component.
++
+.Example output
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+spec:
+  controllerConfig:
+    overrideScheduling:
+      nodeSelector:
+        <key1>: <value1> <1>
+        <key2>: <value2>
+      tolerations:
+        - key: "<key1>" <1>
+          operator: "Equal"
+          value: "<value1>"
+          effect: "NoSchedule"
+  webhookConfig:
+    overrideScheduling:
+      nodeSelector:
+        <key1>: <value1>
+      tolerations:
+        - key: "<key1>"
+          operator: "Equal"
+          value: "<value1>"
+          effect: "NoSchedule"
+  cainjectorConfig:
+    overrideScheduling:
+      nodeSelector:
+        <key1>: <value1>
+      tolerations:
+        - key: "<key1>"
+          operator: "Equal"
+          value: "<value1>"
+          effect: "NoSchedule"
+----
+<1> Replace <key1>, <key2> and <value1> with the specific key-value pairs for `nodeSelector` and `tolerations`.
+
+.Verification
+
+* After applying the updated {cert-manager} custom resource, verify that the scheduling overrides are active:
+
+.. Check the pod specifications for the {cert-manager} components:
++
+[source,terminal]
+----
+$ kubectl get pods -n openshift-cert-manager -o yaml | grep -A 10 "nodeSelector"
```

Comment:
we prefer `oc` over `kubectl`, everywhere!

Reply:
```suggestion
$ oc get pods -n cert-manager -o wide
```
^ also, add an output please!

After which, we can say that "verify if pods were scheduled on the desired nodes"...

Reply:
^ suffices for Verification step 2.